service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,165 @@
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name="placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="Order" displayName="get the person details">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="ordersList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.order.OrderHeader" value-field="ordersList">
+                <field-map field-name="orderId"/>
+            </entity-find-one>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderHeaders" displayName="get the grand_total details">
+        <out-parameters>
+            <parameter name="orderCount" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <set field="orderCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+            </entity-find>
+            <iterate list="ordersList" entry="orderInfo">
+                <if condition="orderInfo.grandTotal &gt; 50">
+                    <then>
+                        <set field="orderCount" from="orderCount+[orderId:orderInfo.orderId,orderName:orderInfo.orderName,
+                        placedDate:orderInfo.placedDate,statusId:orderInfo.statusId,currencyUomId:orderInfo.currencyUomId
+                       ,productStoreId:orderInfo.productStoreId,grandTotal:orderInfo.grandTotal ]"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+
+    <service verb="get" noun="OrderItems" displayName="get the grand_total details">
+        <in-parameters>
+            <parameter name="orderId" default-value="100102"/>
+            <parameter name="orderPartSeqId" default-value="01"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderItemList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId"/>
+                <econdition field-name="orderPartSeqId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderParts" displayName="get the part_total details">
+        <out-parameters>
+            <parameter name="entityCount" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <set field="entityCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+            </entity-find>
+            <iterate list="ordersPartList" entry="entityInfo">
+                <if condition="entityInfo.partTotal &lt; 20">
```

Comment:
The same condition can be used in entity-find, please explore the use of operator attribute.

Reply:
Okay ma'am.

Reply:
Done.